src/views/DownloadPackedOrders.vue

```diff
@@ -59,7 +60,12 @@
             <ion-label>{{ fields[field] ? fields[field].label : field }}</ion-label>
           </ion-item>
         </ion-list>
-      </main>
+      </div>
+      <div class="empty-state" v-else>
+        <img src="../assets/images/emptyState.png" />
+        <p v-html="getErrorMessage()"></p>
```

Comment:
Add the error message directly here, I think there is no need for a method to get the error message.
```suggestion
        <p>{{ translate("There are no packed orders at this facility") }}</p>
```